packages/dmn-js-decision-table/src/features/annotations/components/AnnotationHeader.js

```diff
@@ -1,7 +1,8 @@
-export default function AnnotationHeader() {
+export default function AnnotationHeader(props, context) {
+  const _translate = context.injector.get('translate');
   return (
     <th className="annotation header" rowspan="3">
-      Annotation
+      { _translate('Annotation') || 'Annotation' }
```

Comment:
```suggestion
      { _translate('Annotation') }
```
Translation tool should fall back to the passed string in case of missing translation.

Reply:
Please change this in other places too.